rpc/README.md

```diff
@@ -650,6 +651,51 @@ http://localhost:8114
 }
 ```
 
+### `get_block_economic_state`
+
+Returns increased issuance, miner reward and total transaction fee of a block.
```

Comment:
What's the behavior when the proposal rewards are not finalized yet?

Reply:
[Return `None` if not finalized](https://github.com/nervosnetwork/ckb/pull/1848/files#diff-c80a342630d6634092de83806d174b3eR306-R309), same as [`get_cellbase_output_capacity_details`](https://github.com/nervosnetwork/ckb/tree/dda4ed9/rpc#get_cellbase_output_capacity_details).